### DISCUSSION TITLE
How can I make a custom button?

### DISCUSSION THREAD

**Discussion by PuffedRiceCrackers** [ueberdosis/tiptap#716] [General Questions]
I'd like to make a custom indent button which increases a tag's inline 'margin-left' value.
(ex.`<p>` → `<p style="margin-left:20px;">`) I can refer to existing files (Bold.js.. etc) and documentation, but things are mostly guess work. Two main things I'm curious about are

0. How to detect node based on tag attribute not tag
1. How to manipulate that tag attribute

Thank you for this amazing project :)

**Comment by tonywong625** [comment]
I am also keen on making a custom button/module.

I want to be able to highlight some text then press a button and it will save it into the VueX store to use that piece of text else where
